core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -155,6 +171,27 @@ public synchronized ConfigCheckReport getConfigCheckReport() {
     return mConfigCheckReport;
   }
 
+  /**
+   * Logs the configuration check report information.
+   */
+  public synchronized void logConfigReport() {
+    Status reportStatus = mConfigCheckReport.getStatus();
+    if (reportStatus.equals(Status.PASSED)) {
+      LOG.info("Stauts: {}", reportStatus);
+    } else if (reportStatus.equals(Status.WARN)) {
+      LOG.warn("Status: {}. {}", reportStatus, HELP_INFO);
+      LOG.warn("Warnings: {}", mConfigCheckReport.getConfigWarns().values().stream()
```

Comment:
combine into a single log message - otherwise any number of other log messages could appear between the two separate calls to `LOG.warn`

Reply:
Changed, thanks！